elliottlib/attach_cve_flaws.py

```diff
@@ -1,23 +1,24 @@
-import requests
-import ssl
-from elliottlib import constants, bzutil, errata, util
-from requests_kerberos import HTTPKerberosAuth
+from elliottlib import constants, errata, util
 
 
 def get_attached_tracker_bugs(bzapi, advisory_id):
     return bzapi.getbugs([
         bug['id']
         for bug in get_all_attached_bugs(advisory_id)
-        if is_tracker_bug(bug)
+        if is_tracker_bug(keywords=bug['keywords'])
     ], permissive=False)  # fail if you cannot get all tracker bugs
 
 
 def get_all_attached_bugs(advisory_id):
     return [bug['bug'] for bug in errata.get_raw_erratum(advisory_id)['bugs']['bugs']]
 
 
-def is_tracker_bug(bug):
-    return 'Security' in bug['keywords'] and 'SecurityTracking' in bug['keywords']
+def is_tracker_bug(bug=None, keywords=None):
+    if bug is None and keywords is None:
+        raise ValueError("must pass at least 1 param, either bug object or value of bug keywords")
+    if keywords is None:
+        keywords = bug.keywords
+    return 'Security' in keywords and 'SecurityTracking' in keywords
```

Comment:
to be honest this does not look simpler to me

Reply:
yes it's a weird case of passing bug dict that comes from errata and trying to use the bug obj that comes from bugzilla.. There are ways to convert the dict into object type notations, like namedtuple etc and I wasn't sure if that was simpler. Hmm thinking more about it..

Reply:
Changing the param to keywords only